cinder/files/mitaka/cinder.conf.controller.Debian

```diff
@@ -39,7 +40,7 @@ volume_backend_name=DEFAULT
 
 default_volume_type={{ controller.default_volume_type }}
 
-enabled_backends={% for backend_name, backend in controller.get('backend', {}).iteritems() %}{{ backend_name }}{% if not loop.last %},{% endif %}{% endfor %}
+enabled_backends={% controller.backend|join(',') %}
```

Comment:
I can revert to this ugly code. 
About newton/ocata - it actually requires someone to port kilo/liberty to mitaka/newton/ocata, because config file lacks >300 lines of configuration values.
I don't mind adding **allow_availability_zone_fallback**  to newton/ocata as well, but we should assign someone to port config changes to prior releases.